src/core/public/doc_links/doc_links_service.ts

```diff
@@ -112,7 +112,7 @@ export class DocLinksService {
         kibana: `${ELASTIC_WEBSITE_URL}guide/en/kibana/${DOC_LINK_VERSION}/index.html`,
         siem: {
           guide: `${ELASTIC_WEBSITE_URL}guide/en/security/${DOC_LINK_VERSION}/index.html`,
-          gettingStarted: `${ELASTIC_WEBSITE_URL}guide/en/security/${DOC_LINK_VERSION}/install-siem.html`,
+          gettingStarted: `${ELASTIC_WEBSITE_URL}guide/en/security/${DOC_LINK_VERSION}/index.html`,
```

Comment:
It looks like `install-siem.html` (previously) linked to [Get up and running](https://www.elastic.co/guide/en/security/7.9/install-siem.html), and now it links to what appears to be the default [Elastic Endpoint Security](https://www.elastic.co/guide/en/security/master/index.html) page.

Would you be willing to confirm that we no longer want to link directly to the "getting started" (i.e. `Get up and running`) content?

Reply:
@caitlinbetz for confirmation:

My opinion  - we should link directly to the Endpoint Security integration page.